book/src/design/proving-system/circuit-commitments.md

```diff
@@ -0,0 +1,34 @@
+# Circuit commitments
+
+## Committing to the circuit assignments
+
+At the start of proof creation, the prover has a table of cell assignments that it claims
+satisfy the circuit. The table has $n = 2^k$ rows, and is broken into advice, auxiliary,
+and fixed columns. We define $F_{i,j}$ as the assignment in the $j$th row of the $i$th
+fixed column. Without loss of generality, we'll similarly define $A_{i,j}$ to represent
+the advice and auxiliary assignments.
+
+> The only difference between advice and auxiliary columns, is that the commitments to
+> auxiliary columns are not placed in the proof, and are instead computed by the verifier.
+
+To commit to these assignments, we construct Lagrange polynomials of degree $n$ for each
```

Comment:
these polynomials end up being degree `n - 1` actually; the vanishing polynomial is of degree n though.